packages/dds/tree/src/feature-libraries/flex-tree/lazyEntity.ts

```diff
@@ -38,15 +37,13 @@ export abstract class LazyEntity<TSchema = unknown, TAnchor = unknown>
 	implements FlexTreeEntity<TSchema>, IDisposable
 {
 	readonly #lazyCursor: ITreeSubscriptionCursor;
-	public readonly [anchorSymbol]: TAnchor;
 
 	protected constructor(
 		public readonly context: Context,
 		public readonly schema: TSchema,
 		cursor: ITreeSubscriptionCursor,
-		anchor: TAnchor,
```

Comment:
@CraigMacomber curious about the decision to make `anchor` a "normal" property instead of symbol-indexed. Is there a general guideline we're following to go one way or the other?

Reply:
this is implementing the API in src/feature-libraries/flex-tree/flexTreeTypes.ts. We decided that symbol based APIs are annoying/confusing, so we have been avoiding them in our public API surface, and internally where practical.

When this was not part of that the flex tree API, it had to use a symbol because object nodes reserve all field names not in the API for use a fields, forcing the implementation to keep any internal implemenation specific members of object nodes as JS private or symbols.